e2e-with-targets/02_main_app.R

```diff
@@ -0,0 +1,110 @@
+## app.R
+
+# loading libraries -------------------------------------------------------------------------------
+# if you don't have {box} package yet, install it with:
+# install.packages("box")
+
+# loading packages
+box::use(
+  shiny[...],
+  RSQLite[SQLite],
+  DBI[dbConnect, dbDisconnect, dbGetQuery],
+  glue[glue_sql]
+)
+
+# loading Shiny modules ---------------------------------------------------------------------------
+# box returns crazy error about {rstudioapi} which isn't directly used anywhere
+source("R/app_modules.R", local = TRUE)$value
```

Comment:
Why do you use `source` instead of `box::use(R/app_modules)`? Maybe you need to set option `box.path` in `.Rprofile` so that path is discoverable?